tests/unit/prediction/ocr/test_packager_code.py

```diff
@@ -15,6 +15,9 @@
         ("FR-AB0-123", []),
         ("fr-098-123", []),
         ("Gluten code is FR-234-234 ", ["FR-234-234"]),
+        ("EST  \n 31778", ["EST. 31778"]),
+        ("EST  \n 9999", []),
+        ("M31779+  P31779+  \tV31779", ["M31779 + P31779 + V31779"]),
```

Comment:
We should not have this, instead we should have:
```suggestion
        ("M31779+  P31779+  \tV31779", ["M31779", "P31779", "V31779"]),
```

For this test to pass you will need to change your code.

For more information, see PackagerCodeAnnotator and how emb_code is a list of codes.